Project.toml

```diff
@@ -4,16 +4,22 @@ authors = ["Invenia Technical Computing"]
 version = "0.2.0"
 
 [deps]
-DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
+Missings = "e1d29d7a-bbdc-5cf2-9ac0-f12de2c33e28"
```

Comment:
Where is this used?

I noticed `Vector{Union{Float64, Missing}}(1.0:1.0:20.0)` in the test and was surprised to see `MIssings` doesn't provide a convenience function like `missings(Float64, 1.0:1.0:20.0)`, so i wondered what useful things it does provide?

Reply:
You can use `allowmissing(1.0:1.0:20.0)` for that.

Reply:
Looks like we don't need it anymore.